examples/with-ably/pages/types.d.ts

```diff
@@ -0,0 +1,2 @@
+export type TextMessage = { text: string };
```

Comment:
This file should probably be outside `pages`

Reply:
moved